etc/singularity.conf

```diff
@@ -141,3 +163,14 @@ container dir = /var/singularity/mnt
 # better option.
 #sessiondir prefix = /var/singularity/sessions/
 
+
+# SINGULARITY USER: [STRING]
+# DEFAULT: singularity
+# The protected image mode will restrict Singularity images to those owned by
+# the 'singularity user'; this option allows admins to change the username we
+# use as a test.
+#
+# If user namespaces are enabled, the singularity user setting is ignored and
```

Comment:
@bbockelm What are your thoughts on allowing several `singularity user` lines? The use case for this might be letting user `mibauer` in group `science` bring an image into the cluster that anybody in group `science` could access.